crates/re_viewer_context/src/space_view/space_view_class.rs

```diff
@@ -18,18 +18,18 @@ pub trait SpaceViewClass: std::marker::Sized {
     /// State of a space view.
     type State: SpaceViewState + Default + 'static;
 
-    /// Context of the scene, which is passed to all [`crate::ScenePart`]s and ui drawing on population.
-    type Context: SceneContext + Default + 'static;
+    /// Context of the scene, which is passed to all [`crate::ViewPartSystem`]s and ui drawing on population.
+    type Context: ViewContext + Default + 'static;
 
-    /// Collection of [`crate::ScenePart`]s that this scene populates.
-    type SceneParts: ScenePartCollection<Self> + Default + 'static;
+    /// Collection of [`crate::ViewPartSystem`]s that this scene populates.
+    type ViewPartSystems: ViewPartSystemCollection<Self> + Default + 'static;
```

Comment:
We drop "View" for the other type-definitions.  Also switching from SystemCollection to Systems seems inconsistent.

Maybe this should be:
```suggestion
    type SystemCollection: ViewPartSystemCollection<Self> + Default + 'static;
```